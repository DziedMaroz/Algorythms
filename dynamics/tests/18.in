6507141
354
681884

31748
179
546
663082
478691

4703
946420
2
40685

2
1
914

9
13210
894
92021
1634
2408
156
45
5311
8470
43
02
6491
52
2
57
89
318581
339
280352
6
4
96462
500337

47
12177
97148
8706
647
357524
09416
5337

65
80151
765909
52432
83
880
1
8873
1
557712

4
796684
975047
207
70
28668

5662
9

9511
00
4
8
583483
213396
6188
12
234
833686
4
1706
24562
6
972300
7161
507113

555
545
217
9
961
26


405
32470
130075
29
652519
959
1255
5203

424
055
15679
94996
79
117116
218986
2410
164
72696
4
2773
43
96063
423094
038019
2502
4
8546
651
9
99456
3

95726
8
18125
119
2
61
1237
29331
2898
5
8047
14
3
990
09281
1068

8827
71277
97913
8872
23386
3
71179
33654
25241
8018
170989
02331
05489
03

026570
574464
268
48121
38109
168
05112
66
6968
1
3

16
2414
0
0743
4994
6475
712

21
580618
87116
122030
1197
7795
89
3800
55
097
48090


53032
5681
144431
0

381150

300
560382

472
13286
85
689

671
9
202218
9049
46956

9303
65741
0917
4
9559
61
63658
74175
2482
50
77194
37045
786
9

93745
2
44




8666
062
94888
9840

4
6
1
6821
99653
005
808

295869
54

25
03
463363
46802
4
78183
718827

045
38
258896

153
0
1782
81617
6
00

09
128017
641386

9397
3006
106020
87787
567324
2010
471

25379
640048
4333
719

04
76
4
75
83
846850
4526
50913
43320
4
355
4058
8120
162
0669
530689
57
1932
611
8

002746
0


6
697356

1643



57784
819454
0311
02008
04792
100
7



90985
316
62661
94
99403
84818
34314
97
450706
223968
0
074283
87129
6

4476
93
910571
2
9555
17
840
