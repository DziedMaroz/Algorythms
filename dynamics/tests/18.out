5
6 50 7 14 1
